src/django/api/management/commands/sendtestemail.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand
```

Comment:
I would consider omitting this management command from the PR prior to merging. It is helpful to test this PR, but has limited value later.

If we needed to test email functionality further down the road, I suspect that we would copy and paste an email sending code snippet from the application into a Django shell (to ensure we're replicating what the application does).

Reply:
Agreed. I will remove this prior to merging.